### PR TITLE
Custom jupyter extension docs

### PR DESCRIPTION
**Pull request by marjo-luc**
Contains new/updated documentation for new/revised custom jupyter extensions. Directory structure was modified as well.

### CODE REVIEW

**Comment by marjo-luc** [comment]
@grallewellyn

**Comment by spa0002** [comment]
Should ADE Custom Extensions be beneath Platform Technical Documents like so?
Screenshot 2023-02-08 at 10 04 09 AM
Screenshot 2023-02-08 at 10 09 02 AM

**Comment by marjo-luc** [comment]
This looks like what I intended and what I have locally. Here is an image of the tree from my local build:

local_docs_tree

File: docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "1. Open your workspace\n",
```
Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb:20]: <p>Recommend capitalizing "jupyter"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb:20]: <p>Note: do not see this option on workspace launcher tab (MAAP Basic Stable workspace)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb:20]: <p>Recommend:</p><ul><li>which may -&gt; that may</li><li>indentify -&gt; identify</li><li>Changing instances of "popup" to "pop up"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: docs/source/platform_technical_documentation/ade_custom_extensions/jupyter_server.ipynb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/jupyter_server.ipynb:1]: <p>Recommend:</p><ul><li>Capitalizing instances of "jupyter"</li><li>Changing "endpoints supported" to "supported endpoints"</li><li>Changing "the specific endpoints refer to the" to "the specific endpoints, refer to the"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: docs/source/platform_technical_documentation/ade_custom_extensions/maap_libs.ipynb [hunk 3]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/maap_libs.ipynb:1]: <p>Recommend:</p><ul><li>Capitalizing instances of "jupyter"</li><li>initailizes -&gt; initializes</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/maap_libs.ipynb:1]: <p>Recommend capitalizing instances of "jupyter"</p><p> </p><p>Note:</p><p>When following these instructions, I receive: <code>maap = MAAP(maap_host='api.ops.maap-project.org')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

File: docs/source/platform_technical_documentation/ade_custom_extensions/umf.ipynb [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (spa0002) [inline docs/source/platform_technical_documentation/ade_custom_extensions/umf.ipynb:1]: <p>Recommend changing "user shared" to "user-shared"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb (hunk 1 above)
Comment (marjo-luc) [reply docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb]: This documentation applies to the DIT and UAT venues now, but will also apply to the OPS venue upon the upcoming deployment. Judging by comments in this thread, it looks like the OPS venue was used.

File: docs/source/platform_technical_documentation/ade_custom_extensions/umf.ipynb (hunk 4 above)
Comment (marjo-luc) [reply docs/source/platform_technical_documentation/ade_custom_extensions/umf.ipynb:1]: While I believe the data are user-shared, I want to be consistent with the convention used on the form itself, found [here](https://questionnaire.maap-project.org/). I put the text in bold to better indicate this.

**Review by spa0002 — APPROVED** [review]
Notebook looks good to me.

**Pull request merged**